src/Showpad/Client.php

```diff
@@ -163,13 +163,32 @@ public function assetsTagsAddById($id, $tag)
 
         // Create request
         $data = $this->auth->request(
-            'GET',
+            'LINK',
```

Comment:
Why did you change this? We're using the documented way of using a `POST` request with a `query` parameter containing `method = link` - this works as far as I know (and not all http clients know how to use HTTP commands other than the most used ones...)

Isn't it supported anymore?